### DISCUSSION TITLE
Thermia Itec XT10 ?

### DISCUSSION THREAD

**Discussion by Korsberg** [klejejs/ha-thermia-heat-pump-integration#110] [General]
Can this work with Thermia Itec XT10 Total EQ when is connected via Thermia Online ?

And how is it looking in Home assistant for these Item machines ? can you set/change anty settings from HA ?

Thanks

Peter

**Comment by klejejs** [comment]
Hey! If your heat pump is using Thermia Online, it will work in some capacity. As far as how well it will work and what will be supported, that you have to discover for yourself. 😉 You can install the integration, see what is present there, and if something is missing, open an issue and I will try to fix it.
